catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -709,6 +717,33 @@ public SolrQuery contains(String propertyName, String wkt) {
     return operationToQuery("Contains", propertyName, wkt);
   }
 
+  public static Geometry removeHoles(final Geometry geo) {
```

Comment:
✏️ We should probably log something at a lower level when we remove holes from the given geo.